src/notebook/plugin.ts

```diff
@@ -152,7 +152,14 @@ function activateCellTools(app: JupyterLab, restorer: IInstanceRestorer, tracker
   celltools.addItem({ tool: metadataEditor, rank: 4 });
 
   restorer.add(celltools, namespace);
-  app.shell.addToLeftArea(celltools);
```

Comment:
I think it is a race condition though, and it just happens to work out in the right order.